### ISSUE TITLE
[field] Feedback slots lack aria-live

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#190]
The inputs have feedback slots that have a relationship to the input with `aria-describedby`. This makes it that a user of assistive technology is able to perceive the feedback when focusing on the input. However, sometimes the feedback updates while the user is on the input. For example, when a user is typing, it can get validated instantly. 
`aria-live="politie"`should be added to the feedback slot to have these updates communicated to assistive technology.

**Comment by gerjanvangeest** [comment]
It did, so closing this issue!
